legacy/python/platform/src/actions/install_deps.rs

```diff
@@ -0,0 +1,76 @@
+use moon_action::Operation;
+use moon_common::color;
+use moon_console::{Checkpoint, Console};
+use moon_python_tool::PythonTool;
+use moon_utils::get_workspace_root;
+use std::path::Path;
+
+use crate::find_requirements_txt;
+
+pub async fn install_deps(
+    python: &PythonTool,
+    working_dir: &Path,
+    console: &Console,
+) -> miette::Result<Vec<Operation>> {
+    let mut operations = vec![];
+
+    if let Some(pip_config) = &python.config.pip {
+        let requirements_path = find_requirements_txt(working_dir, &get_workspace_root());
+
+        if let Some(install_args) = &pip_config.install_args {
+            if install_args.iter().any(|x| !x.starts_with("-")) && requirements_path.is_none() {
+                console.out.print_checkpoint(
+                    Checkpoint::Setup,
+                    "Install pip dependencies from install args",
```

Comment:
Change this to just "pip install" to match the other tools

Reply:
Done